infrastructure/resource_group_level/main.bicep

```diff
@@ -276,6 +276,7 @@ module functionAcbs 'modules/function-acbs.bicep' = {
     privateEndpointsSubnetId: vnet.outputs.privateEndpointsSubnetId
     storageAccountName: storage.outputs.storageAccountName
     azureWebsitesDnsZoneId: websitesDns.outputs.azureWebsitesDnsZoneId
+    nodeDeveloperMode: environment == 'dev'
```

Comment:
It didn't seem appropriate to make decisions based on the environment in modules.